__tests__/integration.test.js

```diff
@@ -225,6 +225,22 @@ describe('PATCH: api/articles/:article_id', () => {
     })
 });
 
+describe('DELETE: api/comments/:comment_id', () => {
+    test('204: sends 204 status when comment is deleted', () => {
+        return request(app)
+        .delete('/api/comments/14')
+        .expect(204)
+    });
+    test('404: sends 404 status and message when comment id is invalid', () => {
```

Comment:
This test description is not accurate to the test. This is actually a valid comment id but one that simply does not exist.